prl83/thor.py

```diff
@@ -0,0 +1,159 @@
+# ******************************
+# Power of Thor - Episode 1
+# ******************************
+# -------------
+# Introduction
+# -------------
+# This documentation will walk you through developing a Python 3 solution to the `Power of Thor - Episode 1 puzzle <https://www.codingame.com/ide/puzzle/power-of-thor-episode-1>`_.
+#
+# ------------------
+# Procedure and Tips
+# ------------------
+# So what do I have to do?
+# ====================================
+# .. image:: thor_1.png
+#   :width: 100%
+#
+# The little man is Thor. The red lightning is the "light of power". You have to make Thor walk to the light of power.
+#
+# Sounds easy enough, but what about all that gibberish in the solution window?
+# =======================================================================================
+# .. image:: starter_code.png
+#   :width: 100%
+#
+# The first two lines import libraries that you need.
+import sys
+import math
+# |
```

Comment:
Instead of line blocks, where each line starts with `|`, use better formatting. Here, I'd suggest either a definition list or a bulleted list. For example:

```
# These lines explain the variables given:
#
#   :light_x: the X position of the light of power
#   :light_y: the Y position of the light of power
#   :initial_tx: Thor's starting X position
#   :initial_ty: Thor's starting Y position
#
```

Reply:
This applies to much of your writeup!